plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -454,6 +563,80 @@ protected boolean applyACLRules(Network network, List<? extends InternalIdentity
     }
 
     @Override
+    public boolean applyNetworkACLs(Network config, List<? extends NetworkACLItem> rules) throws ResourceUnavailableException {
```

Comment:
This method never appears to return ``false``.  Why not declare it with as a ``void`` return type.